custom/ewsghana/tasks.py

```diff
@@ -189,6 +190,13 @@ def migrate_email_settings(domain):
     endpoint = GhanaEndpoint.from_config(config)
     migrate_email = EmailSettingsSync(domain)
 
+    for user in iter_docs(WebUser.get_db(), WebUser.ids_by_domain(domain)):
+        for notification in ReportNotification.by_domain_and_owner(domain, user['_id'], stale=False):
+            notification.delete()
```

Comment:
@gcapalbo Not sure if there is another option to stop these emails but this not a problem now because I think that these emails don't populate at all due to fact that ```send_to_me``` property is not set to True.